administrator/components/com_content/models/forms/article.xml

```diff
@@ -706,6 +707,21 @@
 				required="false"
 		        size="1"
 		/>
+
+		<field
+				name="parent_id"
+				type="menuparent"
```

Comment:
If you are expecting dropdown here like this,
![menu_item_parent](https://cloud.githubusercontent.com/assets/2002921/16183456/b04131da-36cf-11e6-8766-5defbfe682d5.png)

Then you will have to include field path for it. `addfieldpath="/administrator/components/com_menus/models/fields"`
For example:
```xml
<fields
			name="menulink" label="COM_CONTENT_MENU_LINK_OPTIONS"
			addfieldpath="/administrator/components/com_menus/models/fields">
```
at line 691.